src/shared/syscheck_op.c

```diff
@@ -906,6 +919,179 @@ const char *get_group(__attribute__((unused)) int gid) {
     return "";
 }
 
+char *get_registry_group(const char *path, char **sid, HANDLE hdnl) {
```

Comment:
`path` is only being used when printing a debug message, could we remove it from the debug message and not pass it?

Reply:
Done ebb1981be1bb1c4b96b4c5658a6dad241a195437